webknossos/tests/test_skeleton.py

```diff
@@ -142,14 +136,13 @@ def test_import_from_nml() -> None:
 
 
 def test_simple_initialization_and_representations(tmp_path: Path) -> None:
-    nml = wk.Skeleton(name="my_skeleton", scale=(0.5, 0.5, 0.5), time=12345)
-    nml_path = tmp_path / "my_skeleton.nml"
+    nml = wk.Skeleton(dataset_name="ds_name", scale=(0.5, 0.5, 0.5))
+    nml_path = tmp_path / "ds_name.nml"
```

Comment:
this could possibly be still my_skeleton.nml?

Reply:
Yes, I search-replaced too much here.